lib/src/component/ref_util.dart

```diff
@@ -47,7 +47,7 @@ import 'package:over_react/component_base.dart';
 ///
 /// Learn more: <https://reactjs.org/docs/refs-and-the-dom.html#creating-refs>.
 Ref<CurrentType> createRef<CurrentType>() {
-  return react_client.createRef<CurrentType>();
+  return react_interop.createRef<CurrentType>();
```

Comment:
update generic parameter to follow [single-letter conventions](https://dart.dev/guides/language/effective-dart/design#do-follow-existing-mnemonic-conventions-when-naming-type-parameters)
```suggestion
  return react_interop.createRef<T>();
```